src/main/java/org/broadinstitute/consent/http/resources/DraftResource.java

```diff
@@ -119,6 +120,23 @@ public Response updateDraft(@Auth AuthUser authUser, @PathParam("draftUUID") Str
     }
   }
 
+  @PATCH
+  @Produces({MediaType.APPLICATION_JSON})
+  @Consumes({MediaType.TEXT_PLAIN})
+  @Path("/v1/{draftUUID}")
+  @RolesAllowed({ADMIN, DATASUBMITTER})
+  public Response patchDraftName(@Auth AuthUser authUser, @PathParam("draftUUID") String draftUUID, String name) {
```

Comment:
Is this the only field in a draft that can be updated? I'm wondering why there's a patch API for name rather than a general API that can update any draft field.

Reply:
Effectively, yes - it's the only metadata field that can be updated.  The created/updated when and by fields are computed by the service.  The payload (e.g. whatever the frontend wants to store as a draft) is handled by the POST endpoint as a replacement for the contents.